benchkit/commandwrappers/trace-cmd.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2023 Huawei Technologies Co., Ltd. All rights reserved.
```

Comment:
@apaolillo, Should I use the vub copyright?

Reply:
indeed, always while you are a VUB student